src/test/java/seedu/address/logic/commands/EventAddVolunteerCommandTest.java

```diff
@@ -67,22 +88,23 @@ public void execute_duplicateVolunteer_throwsCommandException() {
     }
 
     @Test
-    public void execute_validIndexes_addSuccessful() {
-        Model startModel = new ModelManager(getTypicalEventStorage(), getTypicalVolunteerStorage(),
-                new UserPrefs());
-        Index validEventIndex = Index.fromOneBased(startModel.getFilteredEventList().size());
-        Index validVolunteerIndex = Index.fromOneBased(2);
-        EventAddVolunteerCommand command = new EventAddVolunteerCommand(validEventIndex, validVolunteerIndex);
+    public void execute_clashingEvent_throwsCommandException() {
+        ObservableList<Event> eventList = model.getEventStorage().getEventList();
+        ObservableList<Volunteer> volunteerList = model.getVolunteerStorage().getVolunteerList();
+        // Assign a volunteer to an existing event
```

Comment:
i think this one happened for my vliste also, but can consider breaking this line into smaller portions 
(e.g. `Event event = ... `
`Volunteer volunteerToAssign = ...`
`event.addVolunteer(volunteerToAssign)`

Reply:
aight